docs/index.rst

```diff
@@ -1,19 +1,19 @@
-Confit: Painless Configuration
+Confuse: Painless Configuration
 ==============================
 
-`Confit`_ (*con-FEE*) is a straightforward, full-featured configuration system
+`Confuse`_ (*con-FEE*) is a straightforward, full-featured configuration system
```

Comment:
I'm not sure if the pronunciation would stay as *con-FEE*, as that isn't anywhere near how you pronounce Confuse 😆.